lib/db.py

```diff
@@ -68,6 +68,17 @@ def fetch_one(self, query, params=None, fetch_mode=None):
     def fetch_all(self, query, params=None, fetch_mode=None):
         return self.__query(query, params=params, return_type='all', fetch_mode=fetch_mode)
 
+    def import_csv(self, filename):
+        with self._pool.connection() as conn:
+            conn.autocommit = True
+            cur = conn.cursor()
+            with open(filename, 'r', encoding='utf-8') as sql_file:
+                sql_script = sql_file.read()
+                for statement in sql_script.split(';'):
```

Comment:
This will fail if there is a `;` in a text value for example

Reply:
hmm, ja. That is an issue. It worked fine for my testing but definitely has to be reworked. Since I do not use that function anymore as I choose a different implementation to ingest the data I added a NotImplementedError Guard clause for now